src/ui/components/Rating/index.js

```diff
@@ -95,7 +95,7 @@ export class RatingBase extends React.Component {
     let description;
     if (rating) {
       description = i18n.sprintf(i18n.gettext('Rated %(rating)s out of 5'),
-        { rating: i18n.formatNumber(parseFloat(rating.toFixed(1))) });
+        { rating: i18n.formatNumber(parseFloat(rating).toFixed(1)) });
```

Comment:
This seems like the right thing to do but can you add a test for it? I didn't see one.

Reply:
Will do! @diox helpfully patched the API response but we still feed integers to it in some of the tests and also it just makes sense to do. I'll add it. 👍